javascript/stimulus_reflex.js

```diff
@@ -1,7 +1,9 @@
 import ActionCable from 'actioncable';
 import CableReady from 'cable_ready';
 import StimulusReflexController from './stimulus_reflex_controller';
+import { dispatch } from './helpers';
```

Comment:
Is helpers the best way we can define this? Maybe just creating a file called dispatch? Since it’s only included in one file it feels like helpers is the wrong namespace to put this but I don’t hate it. Just want it to make sense for other devs reading this source code.

Reply:
Valid nitpick. Will think through this a bit more.